src/Header.js

```diff
@@ -0,0 +1,74 @@
+import logo from "./the-logo.png";
+import styled from "styled-components";
+import { FaShoppingCart } from "react-icons/fa";
+
+  const TopNav = styled.nav`
+    display: flex;
+    padding: 0.2em 0.2em;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 1.2em;
+    background-color: #94b053;
+    ul {
+      list-style: none;
+      display: flex;
+      justify-content: flex-end;
+    }
+
+    li {
```

Comment:
It's preferable to create a 'styled component' for each building block that we have in our app rather than create nested styling. This way we don't depend on the arrangement of our blocks or their class for styles to be applied correctly and we can share these components across the application i.e `TextInput` 
```js
const TopNav = styled.nav`
  /* Your nav styles here */`

const NavItemsContainer = styled.ul`
/* Your ul styles here */
`
const NavItems = styled.li`
/* Your li styles here */
`
const TextInput = styled.input`
/* Your input styles here */
`

```
then in our component 
```js
<TopNav>
        <LogoHeader href="./">
          <img src={logo} alt="logo" />
          </LogoHeader>

        <NavItemsContainer>
          <NavItem>
            <TextInput
              disabled
              placeholder="Looking for..."
            />
          </NavItem>
          <NavItemsContainer>
</TopNav>

```